app/models/consignment_inquiry.rb

```diff
@@ -0,0 +1,5 @@
+class ConsignmentInquiry < ApplicationRecord
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+  validates :name, presence: { message: "is required" }
+  validates :message, presence: { message: "is required" }
```

Comment:
**thought:** is it possible to check for the trim here?

Reply:
no length check is required, i am referring to avoiding spaces mainly here

Reply:
Oh. Ok. Yeah, `presence:` does not allow spaces. For eg `message: "-----------space -----------"` will fail validation.